fuzz/CMakeLists.txt

```diff
@@ -18,6 +18,10 @@ set(FUZZ_LINKER_LANGUAGE "C" CACHE STRING "Linker language for fuzz harnesses")
 mark_as_advanced(FUZZ_LINKER_LANGUAGE)
 enable_language(${FUZZ_LINKER_LANGUAGE})
 
+if(NOT DEFINED FUZZ_LDFLAGS)
```

Comment:
Would `if(NOT FUZZ_LDFLAGS)` make more sense here? I'm pretty sure an empty `FUZZ_LDFLAGS` will be problematic as well.